haskell/cabal.bzl

```diff
@@ -212,6 +212,15 @@ def _prepare_cabal_inputs(
         for arg in ["-package-db", "./" + _dirname(package_db)]
     ], join_with = " ", format_each = "--ghc-arg=%s", omit_if_empty = False)
     args.add("--flags=" + " ".join(flags))
+    if not hs.toolchain.is_darwin and not hs.toolchain.is_windows:
+        # On some more recent Linux distributions (e.g. Ubuntu 18.04) `gcc`
```

Comment:
I'd suggest using [notes](https://ro-che.info/articles/2018-05-02-ghc-note-snippet) to link these comments together, and include a link to the original ticket.

Reply:
Good idea, I've added a more detailed note and pointed to it next to the changes.